front/Dockerfile

```diff
@@ -30,26 +30,27 @@ FROM base AS elixir
 WORKDIR /elixir
 RUN mix local.hex --force \
     && mix local.rebar --force
-COPY mix.* ./
+COPY front/mix.* ./
+COPY feature_provider ../feature_provider
 RUN --mount=type=ssh mix do deps.get, deps.compile
```

Comment:
🤔 I think we can get rid of the `--mount=type=ssh` now that the feature provider is part of the bundle.